### PR TITLE
Bug Fix: pretrained s3prl-frontend based models loaded with parameters key mismatch error

### PR DESCRIPTION
**Pull request by simpleoier**
This is related to #4910 .
Thanks @boeddeker for the solution!

### CODE REVIEW

**Comment by sw005320** [comment]
Can you fix some CI errors?
Then, I'll merge it.

**Comment by sw005320** [comment]
Thanks, @simpleoier!
@boeddeker, could you test it?

File: espnet2/tasks/abs_task.py [hunk 1]
```diff
@@ -1875,6 +1875,30 @@ def build_model_from_file(
                 # NOTE(kamo): "cuda" for torch.load always indicates cuda:0
                 #   in PyTorch<=1.4
                 device = f"cuda:{torch.cuda.current_device()}"
-            model.load_state_dict(torch.load(model_file, map_location=device))
+            try:
+                model.load_state_dict(torch.load(model_file, map_location=device))
```
Comment (boeddeker) [inline espnet2/tasks/abs_task.py:1879]: Could you change this to 

```python
state_dict = torch.load(model_file, map_location=device)
try:
    model.load_state_dict(state_dict)
```
and remove the later call of `torch.load`?

**Comment by boeddeker** [comment]
Sorry for the late reply. Yes, the code works to load an old model. Thanks.
The downside is just, that there is now a file system overhead, because the file is loaded multiple times.

Could that be fixed? (See my comment in the code)

**Pull request merged**